notes/force-move-spec.md

```diff
@@ -0,0 +1,208 @@
+# Force Move Spec
+
+To serve as a succinct specification of the ForceMove protocol.
+Everthing required for getting to an outcome.
+
+Things have changed since the original paper.
+Names have changed. Optimizations have been added.
+This should serve as the definitive version over time.
+
+Not a concrete specification.
+Abstract notion of "the state of the channel".
+Don't specify formats.
+In methods, we provide the minimal required parameters.
+In a concrete implementation, the parameters might change - for example it can be more efficient
+
+## Off-chain state
+
+| **Field**      | **Data type**                 | **Definition / Explanation**                                                                         |
+| :------------- | :---------------------------- | :--------------------------------------------------------------------------------------------------- |
+| ChainID        | `bytes32`                     | e.g. ropsten / mainnet                                                                               |
+| Participants   | `address[]`                   | addresses of participants. Determines keys used to sign updates                                      |
+| ChannelNonce   | `uint256`                     | unique, to prevent replay attacks                                                                    |
+| TurnNum        | `uint256`                     | turn number                                                                                          |
+| DefaultOutcome | `[address, uint256][]`        | tracks the amounts paid out to each of a list of addresses if the channel is finalized in this state |
```

Comment:
Where is the discussion recorded on the data type of this?

Reply:
Nowhere yet. Probably should change this to `bytes` here.